File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int i, j;// initializing required variables
    int height;

    do
    {
        height = get_int("enter the height\n");     // asking height from the user
    }
    while (height < 1 || height > 8);

    for (i = 0; i < height; i++)// loop to change the line
    {
        for (j = 0; j < height; j++)//loop to print left pyramid
        {
            if (j >= height - 1 - i)
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }

        printf("  ");       // to print thr gap btwn to pyramids

        for (j = 0; j <= i; j++) //to print the right pyramid
        {
            printf("#");
        }

        printf("\n");

    }
}

File: mario.py
from cs50 import get_int

while True:
    x = get_int("Height: ")
    if x >= 1 and x <= 8:
        break
for i in range(x):
    for j in range(x + 3 + i):
        if j == x + 1:
            print(" ", end="")
        elif j >= x - i:
            print("#", end="")
        else:
            print(" ", end="")
    print()

